src/group_theory/commutator.lean

```diff
@@ -21,7 +21,19 @@ is the subgroup of `G` generated by the commutators `h₁ * h₂ * h₁⁻¹ * h
 * `⁅H₁, H₂⁆` : the commutator of the subgroups `H₁` and `H₂`.
 -/
 
-variables {G G' F : Type*} [group G] [group G'] [monoid_hom_class F G G'] (f : F) (g₁ g₂ g₃ : G)
+variables {G G' F : Type*} [group G] [group G'] [monoid_hom_class F G G'] (f : F) (g₁ g₂ g₃ g : G)
+
+lemma commutator_element_eq_one_iff_mul_comm : ⁅g₁, g₂⁆ = 1 ↔ g₁ * g₂ = g₂ * g₁ :=
+by rw [commutator_element_def, mul_inv_eq_one, mul_inv_eq_iff_eq_mul]
+
+lemma commutator_element_eq_one_iff_commute : ⁅g₁, g₂⁆ = 1 ↔ commute g₁ g₂ :=
+commutator_element_eq_one_iff_mul_comm g₁ g₂
```

Comment:
I think it's worth adding a `commute.commutator_eq` alias for the backwards direction of this statement.

Then you can use it to prove the two lemmas below as `(commute_one_left g).commutator_eq` etc.